src/assets/scss/_comment.scss

```diff
@@ -0,0 +1,87 @@
+.comments {
+  width: 72%;
+  display: flex;
+  flex-direction: column;
+  position: relative;
+  left: 23%;
+}
+
+.overlap {
+  display: grid;
+  grid-template-areas: 'comment';
+}
+
+.typeComment {
```

Comment:
This css class name breaks the convention we have on the codebase. Avoid using camelcase for css class names